source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst

```diff
@@ -55,3 +55,25 @@ is supported on platforms:
 		root@am62xx-evm:~# modprobe ili210x
 
 After re-initializing the module, the touch IC should now be registered as an input and touch should be functional.
+
+**Note**: The following steps are only applicable on AM625 SK EVM revision B/B-P1 board
```

Comment:
This is not the way device specific documentation should be done. There are clear sections in this documentation that are not using the section notation. Notes are not a good way to indicate a platform switch.

Reply:
Created sections for different revisions.